utilities/bin/core/commands.dsc

```diff
@@ -0,0 +1,503 @@
+##IgnoreWarning command_script_usage
+##IgnoreWarning tag_trace_failure
+##IgnoreWarning bad_tag_part
```

Comment:
Both of these may be removed if the below tags are resolved
```suggestion
```

Reply:
Already fixed.